src/preProcessPattern.js

```diff
@@ -39,9 +38,13 @@ export default function preProcessPattern(globalRef, pattern) {
     debug(`determined '${pattern.to}' is a '${pattern.toType}'`);
 
     // If we know it's a glob, then bail early
-    if (_.isObject(pattern.from) && pattern.from.glob) {
+    if (Object.prototype.toString.call(pattern.from) === '[object Object]' && pattern.from.glob) {
```

Comment:
`typeof pattern.from === 'object'` ?

Reply:
@michael-ciniawsky `typeof null === 'object'` and `typeof [1,2] === 'object'`
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof

Reply:
Could you move this to into a helper function `isObject` instead please ? 

```js
const isObject = (val) => Object.prototype.toString.call(val) === '[object Object]' ? true : false
```